### ISSUE TITLE
APNG support

### ISSUE THREAD

**Issue by commandblockguy** [discord/lilliput#101]
Currently, lilliput removes all of the APNG-specific data when resizing an animated PNG file, causing it to only output the first frame of the animation. Ideally, it should resize and output the remainder of the APNG file as well.

**Comment by BitterSweetcandyshop** [comment]
ah yes I comment to support this basic and  should already exsit pr... (Please add this

**Comment by Knewest** [comment]
When they addressed the 'aCropalypse' vulnerability, they completely killed APNG. 
They are now automatically converted to static PNGs. We need APNG support more than ever now.

**Comment by DalekCraft2** [comment]
> When they addressed the 'aCropalypse' vulnerability, they completely killed APNG.

I am not knowledgeable about the "acropalypse", but I have heard that animated WebP used to be supported before they removed the support for it; did that happen around the same time as the APNG support removal?

**Comment by Knewest** [comment]
> ... did that happen around the same time as the APNG support removal?

No, animated WebP has not been "supported" for a very long time. It is supported, Discord just chooses not to embed it, like with BMP and AVIF, to name a few.

**Comment by M0chaCat** [comment]
is there any update on APNG support now that its officially part of the PNG spec

**Comment by kiki-ala** [comment]
Apologies; while the APNG looks better I saw that I can just upload GIF.

**Comment by M0chaCat** [comment]
hm? also is your username toki pona?

**Comment by kiki-ala** [comment]
I had left a comment admonishing them for hiring nothing but "ai engineers" the past three years while discord has the same inexplicable bugs that never improve. And i got rejected for every application so i was blaming people at discord for my lack of work, since clearly the bugs are there, i would fix them, but they're idiots

Because i couldn't upload an apng that fit the file specs. My issue likely has little to do with liliput


Then i just uploaded a gif and edited my comment because who gives a shit

But i'm still mad at them so here you go

Also no idea what that is
